pdq/cpp/hashing/pdqhashing.cpp

```diff
@@ -11,6 +11,11 @@
 // https://github.com/facebookexternal/ThreatExchange-PDQ/blob/main/pdqhash-2017-10-09.pdf
 // ================================================================
 
+#if defined(_WIN32)
+  #define _USE_MATH_DEFINES
+  #include <iostream>
```

Comment:
(Dumb question, it's been a long time since cpp): I don't see cin/cout or friends anywhere, did you include this for functionality or to help with debugging? (Same question on other imports)